app.js

```diff
@@ -25,20 +25,25 @@ module.exports = {redis: r};
 // Initialize logger
 const log4js = require('log4js');
 log4js.configure({
-	'appenders': [
-		{
+	appenders: {
+		out: {
 			type: 'console',
 			category: 'purdue-laundry'
 		},
-		{
-			type: 'loggly',
-			token: '5ac059d3-1b8c-4e5e-9059-0b6b080b3fee',
-			subdomain: 'purduesigapp',
-			tags: ['purdue-laundry'],
-			category: 'purdue-laundry',
-			json: true
-		},
-	]
+		// Commenting out loggly stuff since it doesn't work and I have no idea what loggly account it goes to
```

Comment:
lol i have no idea where this goes to either. honestly maybe one day ill just get rid of all of this and go all in on GA